File: examples-sdk/csharp/036-basic-sse-event-builder/Main.cs
// 036-basic-sse-event-builder — C# SDK equivalent
// Compile: vil compile --from csharp --input 036-basic-sse-event-builder/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("stock-market-ticker", 8080);
var ticker = new ServiceProcess("ticker");
ticker.Endpoint("GET", "/stream", "ticker_stream");
ticker.Endpoint("GET", "/info", "ticker_info");
server.Service(ticker);
server.Compile();

File: examples-sdk/swift/036-basic-sse-event-builder/main.swift
// 036-basic-sse-event-builder — Swift SDK equivalent
// Compile: vil compile --from swift --input 036-basic-sse-event-builder/main.swift --release

let server = VilServer(name: "stock-market-ticker", port: 8080)
let ticker = ServiceProcess(name: "ticker")
ticker.endpoint(method: "GET", path: "/stream", handler: "ticker_stream")
ticker.endpoint(method: "GET", path: "/info", handler: "ticker_info")
server.service(ticker)
server.compile()
